### PR TITLE
Check for 'null' in getWinOrigin

### PR DESCRIPTION
**Pull request by camelburrito**
Check for 'null' in getWinOrigin

### CODE REVIEW

File: src/url.js [hunk 1]
```diff
@@ -54,7 +54,8 @@ export const SOURCE_ORIGIN_PARAM = '__amp_source_origin';
  * @return {string} origin
  */
 export function getWinOrigin(win) {
-    return win.origin || parseUrl(win.location.href).origin;
+  return (
+    win.origin || win.origin==-"null" || parseUrl(win.location.href).origin);
```
Comment (dvoytenko) [inline src/url.js]: I'm not quite understanding what this means. Please clarify. Why `==-` (minus in the end)?

Comment (camelburrito) [reply src/url.js]: lol - just fixed this .

Comment (camelburrito) [reply src/url.js]: Done

**Comment by camelburrito** [comment]
@dvoytenko - i think my amend ate your comment - can you PTAL

**Comment by jridgewell** [comment]
Does this have a bug report it should close?

**Comment by dvoytenko** [comment]
@camelburrito there might be one in AMP Start?

**Pull request merged**